src/test/java/org/fundacionjala/coding/richard/severich/spinwords/SpinWordsTest.java

```diff
@@ -0,0 +1,41 @@
+package org.fundacionjala.coding.richard.severich.spinwords;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by RichardSeverich on 5/22/2017.
+ */
+public class SpinWordsTest {
+
+    /**
+     * This test checks the string that returns the method getSpinWord returns the expected string.
+     */
+    @Test
+    public void verifyChangeHeyFellowWarriors() {
+        SpinWords spinWords = new SpinWords();
+        String expectedResult = "Hey wollef sroirraw";
+        assertEquals(expectedResult, spinWords.spin("Hey fellow warriors"));
+    }
+
+    /**
+     * This test checks the string that returns the method getSpinWord returns the expected string.
+     */
+    @Test
+    public void verifyChangeThisIsATest() {
+        SpinWords spinWords = new SpinWords();
+        String expectedResult = "This is a test";
+        assertEquals(expectedResult, spinWords.spin("This is a test"));
+    }
+
+    /**
+     * This test checks the string that returns the method getSpinWord returns the expected string.
+     */
+    @Test
+    public void verifyChangeThisIsAnotherTest() {
+        SpinWords spinWords = new SpinWords();
```

Comment:
You are repeating this code in all your tests
Please put them in @before method or convert it to a static method.